.taskcluster.yml

```diff
@@ -248,6 +250,7 @@ tasks:
                                           MOBILE_REPOSITORY_TYPE: git
                                           MOZ_AUTOMATION: "1"
                                           REPOSITORIES: {$json: {mobile: "firefox-android"}}
+                                          TASKCLUSTER_CACHES: /builds/worker/.gradle
```

Comment:
@jcristau and I found this environment variable earlier today. It lets `run-task`[1] mount the cache with the right owner and permissions. 

[1] https://github.com/taskcluster/taskgraph/blob/3b143714d08019f56d6fc98a8b34c6bafc314fd9/src/taskgraph/run-task/run-task#L1110-L1111